static/js/admin.js

```diff
@@ -129,19 +129,38 @@ exports.populate_emoji = function (emoji_data) {
     var emoji_table = $('#admin_emoji_table').expectOne();
     emoji_table.find('tr.emoji_row').remove();
     _.each(emoji_data, function (data, name) {
-        emoji_table.append(templates.render('admin_emoji_list', {emoji: {name: name, source_url: data.source_url,
-                                                                         display_url: data.display_url}}));
+        emoji_table.append(templates.render('admin_emoji_list', {
+            emoji: {
+                name: name, source_url: data.source_url,
+                display_url: data.display_url
+            }
+        }));
     });
     loading.destroy_indicator($('#admin_page_emoji_loading_indicator'));
 };
 
+exports.populate_auth_methods = function (auth_methods) {
+    var auth_methods_table = $("#admin_auth_methods_table").expectOne();
+    auth_methods_table.find('tr.method_row').remove();
+    _.each(auth_methods, function (val,key) {
+        auth_methods_table.append(templates.render('admin_auth_methods_list', {
+            method: {
+                method: key,
+                enabled: val
+            }
+        }));
+    });
+    loading.destroy_indicator($('#admin_page_auth_methods_loading_indicator'));
+};
+
 function _setup_page() {
     var options = {
-        realm_name:                 page_params.realm_name,
-        domain:                     page_params.domain,
+        realm_name: page_params.realm_name,
+        domain: page_params.domain,
         realm_restricted_to_domain: page_params.realm_restricted_to_domain,
-        realm_invite_required:      page_params.realm_invite_required,
+        realm_invite_required: page_params.realm_invite_required,
```

Comment:
This cleanup is great, but I'd probably make it its own commit at the start of the branch so we can merge it independently of the feature, since it's not really part of the new feature